### PR TITLE
git-octopus 1.1 (updated formula)

### PR DESCRIPTION
**Pull request by mathieubolla**
Updated to release 1.1
Removes HEAD (build system too unstable)

### CODE REVIEW

File: Library/Formula/git-octopus.rb [hunk 1]
```diff
@@ -1,21 +1,16 @@
 class GitOctopus < Formula
   desc "Extends git-merge with branch naming patterns"
   homepage "https://github.com/lesfurets/git-octopus"
-  url "https://github.com/lesfurets/git-octopus/archive/v1.0.tar.gz"
-  sha256 "c83ab4aa3d8ec770f4f05619239a7852c871216a108b4e287c13efb10e3db5db"
-
-  head "https://github.com/lesfurets/git-octopus.git"
-
-  bottle do
```
Comment (DomT4) [inline Library/Formula/git-octopus.rb]: Please leave this block intact. We'll refresh the bottles on pull automatically.

File: Library/Formula/git-octopus.rb [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
-    sha256 "7c953c0d60e50045229f63d9efecb9a9b25791f336efde09b91026067102682e" => :mountain_lion
-  end
+  url "https://github.com/lesfurets/git-octopus/archive/v1.1.tar.gz"
+  sha256 "cc0f64be23a385a18e8cd7e913a54e582f9c8c07460b2c50b0df46b83fc3769d"
 
   def install
+    system "make", "build"
     bin.install "bin/git-octopus"
-    man1.install "man/man1/git-octopus.1"
+    bin.install "bin/git-conflict"
```
Comment (DomT4) [inline Library/Formula/git-octopus.rb]: You can put this on one line. `bin.install "bin/git-conflict", "bin/git-apply-conflict-resolution"`

Same with the `man1` lines below.

**Comment by mathieubolla** [comment]
Ok, I've amended my branch taking your comments into account. Thank you for your input. Is ok now?

**Comment by DomT4** [comment]
Looks great, Thanks! Will let our CI go over it and then we'll pull, presuming all is okay.

**Comment by mikemcquaid** [comment]
git-octopus 1.1

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `git-octopus 1.1`.

**Pull request closed without merging**